src/sitewiseCache.ts

```diff
@@ -117,20 +118,16 @@ export class SitewiseCache {
   }
 
   async getModelsOptions(): Promise<Array<SelectableValue<string>> | undefined> {
-    const options = getTemplateVariableOptions();
-
     const models = await this.getModels();
     if (!models) {
       return;
     }
 
-    return options.concat(
-      models.toArray().map((model) => ({
-        label: model.name,
-        value: model.id,
-        description: model.description,
-      }))
-    );
+    return models.toArray().map((model) => ({
```

Comment:
(we could probably move this function getModelsOptions back to the component in the refactor you mentioned)

Reply:
yes exactly the plan, trying not to turn this PR into the "big cache refactor" but it will come